standard/types.md

```diff
@@ -547,7 +547,7 @@ Each type argument shall satisfy any constraints on the corresponding type param
 
 ### 8.4.3 Open and closed types
 
-Any type can be classified as either an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
+All types can be classified as either ***open types*** or ***closed types***. An open type is a type that involves type parameters. More specifically:
```

Comment:
The “can be” here should be something like “are” – we are defining things here. #1441 changes this line to:

```suggestion
Any type can be classified as either an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

This still says “can be” so I think it is wrong. It also reads wrong to me, but that is subjective. I’m going to make some alternative suggestions, each in a seperate comment. Please 👍/👎🏻 as you see fit, including this one, and whomever merges can OK the preferred choice, or we don't merge and #1441 stands.

Reply:
Is “Every” better than “A”?

```suggestion
Every type is classified as an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

Reply:
I prefer "A type is either an ***open type*** or a ***closed type***. " for 2 reasons:

1. When defining a term it's best to use the singular.
2. I prefer "type" as part of the term being defined, as without it the terms ***open*** and ***closed*** are just adjectives with no context.